docs/linkify.rst

```diff
@@ -26,9 +26,22 @@ having URLs in tag attributes getting linkified.
    You may pass a ``string`` or ``unicode`` object, but Bleach will always
    return ``unicode``.
 
+.. note::
 
-.. autofunction:: bleach.linkify
+   `linkify` **does not** attempt to protect users from bad or
```

Comment:
Does it make it clearer to say "by default" here thereby distinguishing Bleach's responsibilities from developer's ability to extend/augment the default behavior using callbacks?

Reply:
Yeah good call! Added "by default". We should link to examples / docs / extras functions that handle the homograph attacks once they exist.